x-pack/plugin/ml/qa/multi-cluster-tests-with-security/src/test/resources/rest-api-spec/test/multi_cluster/50_sparse_vector.yml

```diff
@@ -0,0 +1,184 @@
+# This test uses the simple model defined in
+# TextExpansionQueryIT.java to create the token weights.
+setup:
```

Comment:
What is the need to add `sparse_vector` query to the multicluster test?

Reply:
I added this retroactively for the `text_expansion` query after we broke the CCS compat check during a commit. While we caught the regression before it was released, it felt like a good idea to add support for multi cluster testing of some sort.

Reply:
Thanks for the clarification, Kathleen!